onnx/version_converter/convert.h

```diff
@@ -606,6 +606,12 @@ class DefaultVersionConverter : public BaseVersionConverter {
         std::make_unique<TypeRestriction>("ReduceMax", OpSetID(20), OpSetID(19), reduce_min_max_18_unallowed_types));
     registerAdapter(
         std::make_unique<TypeRestriction>("ReduceMin", OpSetID(20), OpSetID(19), reduce_min_max_18_unallowed_types));
+
+    /******** 20 -> 21 ********/
+    registerAdapter(std::make_unique<CompatibleAdapter>("QLinearMatMul", OpSetID(20), OpSetID(21)));
+
+    /******** 21 -> 20 ********/
+    registerAdapter(std::make_unique<CompatibleAdapter>("QLinearMatMul", OpSetID(21), OpSetID(20)));
```

Comment:
This should restrict the conversion to the types supported in earlier version. Eg., see [here](https://github.com/onnx/onnx/blob/1784aeee3a8a9c819e81e516048475dec1d545fd/onnx/version_converter/convert.h#L602)

Reply:
Hmmm .. except that the old linked example above has the version-numbers swapped (which is a typo/bug that should be fixed, I guess).

Reply:
done